src/package.xml

```diff
@@ -1190,7 +1190,13 @@
         <members>npe01__OppPayment__c.Payment_Acknowledgment_Status__c</members>
         <members>npe03__Recurring_Donation__c.Always_Use_Last_Day_Of_Month__c</members>
         <members>npe03__Recurring_Donation__c.Day_of_Month__c</members>
-        <members>npe03__Recurring_Donations_Settings__c.Recurring_Donation_Batch_Size__c</members>
+        <members>npe03__Recurring_Donation__c.InstallmentFrequency__c</members>        
+        <members>npe03__Recurring_Donation__c.PaymentMethod__c</members>
+        <members>npe03__Recurring_Donation__c.RecurringType__c</members>
+        <members>npe03__Recurring_Donation__c.Status__c</members>
+        <members>npe03__Recurring_Donations_Settings__c.IsRecurringDonations2Enabled__c</members>
+        <members>npe03__Recurring_Donations_Settings__c.Recurring_Donation_Batch_Size__c</members>        
+        <members>npe03__Recurring_Donations_Settings__c.InstallmentOppStageName__c</members>
```

Comment:
This line should move up one so the fields are in alphabetical order for the object.

Reply:
The changes are done and ready for review, thanks, @force2b !